test/parallel/test-http-response-splitting.js

```diff
@@ -19,23 +19,26 @@ const y = 'foo⠊Set-Cookie: foo=bar';
 
 let count = 0;
 
+function test(res, code, header) {
+  const errRegExp = new RegExp(
+    'TypeError: The header content contains invalid characters'
```

Comment:
Can you add `^` and `$` to the regular expression. You can also use a regex literal instead of the `RegExp()` constructor.